docker-compose.yml

```diff
@@ -5,4 +5,4 @@ dnsdock:
     volumes:
         - /var/run/docker.sock:/run/docker.sock
     ports:
-        - 172.17.42.1:53:53/udp
+        - 53:53/udp
```

Comment:
please remove this, this is platform dependence. This will works only at linux or not?